scss/_variables.scss

```diff
@@ -1644,3 +1644,10 @@ $back-to-top-icon-width:         add(.5rem, 1px) !default;
 $back-to-top-icon-height:        subtract(1rem, 1px) !default;
 // scss-docs-end back-to-top
 // End mod
```

Comment:
This comment should stay at the end of the file, preferably (stepped process is still a Boosted mod).